AirCasting/CoreData/Storages/SessionStorage.swift

```diff
@@ -1,40 +1,53 @@
-// Created by Lunar on 27/05/2021.
+// Created by Lunar on 09/08/2022.
 //
-
-import Foundation
 import CoreData
 import Resolver
 
-final class SessionDataEraser: ObservableObject {
-    @Injected private var persistenceController: PersistenceController
+protocol SessionStorage {
```

Comment:
I think this should not be called just "SessionStorage" if it has just one function which is related to clearing bluetooth connection. The name of this protocol should be more specific